package.json

```diff
@@ -37,9 +37,10 @@
     "babel-core": "^6.2.1",
     "babel-eslint": "^7.1.1",
     "babel-loader": "^6.2.4",
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
```

Comment:
this is like `babel-polyfill`, except it's more automatic and more  configurable. I include it only to support generators, to support async/await.